games-puzzle/pingus/files/pingus-0.7.6-gcc15-fix.patch

```diff
@@ -0,0 +1,23 @@
+From: OldManSeph7818 <sschaefering@gmail.com>
+Date: Wed, 29 Jan, 2025 12:21
+Subject: [PATCH] fix gcc-15 build
+
+--- a/src/engine/display/font.hpp
++++ b/src/engine/display/font.hpp
+
+@@ -22,2 +22,3 @@
+-#include "math/rect.hpp"
```

Comment:
For some reason the diff is bigger than needed as it replaces this line by the same one. On the other hand, Fedora patch properly adds only the minimum needed lines:
https://src.fedoraproject.org/rpms/pingus/blob/rawhide/f/includes.patch

Maybe that patch could be used instead 

Thanks!